packages/cli/src/playground/src/lib/analytics.tsx

```diff
@@ -0,0 +1,22 @@
+import posthog from 'posthog-js';
+import { PostHogProvider as PHProvider } from 'posthog-js/react';
+import { useEffect } from 'react';
+
+export function PostHogProvider({ children }: { children: React.ReactNode }) {
+  useEffect(() => {
+    if (import.meta.env.VITE_NO_MASTRA_TELEMETRY) {
+      console.info('[Analytics]: Telemetry is disabled.');
+      return;
+    }
+
+    posthog.init('phc_SBLpZVAB6jmHOct9CABq3PF0Yn5FU3G2FgT4xUr2XrT', {
```

Comment:
Should we capture the folder name or git repo, etc, as well? If I have five projects we won't be able to distinguish.

https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-telemetry/src/repository-id.ts

Reply:
Is that possible to forward data from the CLI to the vite process?